curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/5ddb965c65d27e1512d44d9a.md

```diff
@@ -52,8 +52,22 @@ assert.equal(document.querySelector('form')?.id, 'calorie-counter');
 ```
 
 ```css
+:root {
+  --light-text: #f5f6f7;
+  --dark-text: #0a0a23;
+  --fcc-blue: #1b1b32;
+  --light-yellow: #fecc4c;
+  --dark-yellow: #feac32;
+  --light-pink: #ffadad;
+  --dark-red: #850000;
+  --light-green: #acd157;
+}
+
 body {
-  font-family: "Lato", Arial;
+  font-family: "Lato", Helvetica, Arial, sans-serif;
+  font-size: 18px;
+  background-color: var(--fcc-blue);
+  color: var(--light-text);
```

Comment:
If you do decide to update the variable name as suggested above then it will need to be updated here too